src/App.tsx

```diff
@@ -15,48 +16,107 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+type SortType = 'NONE' | 'ALPABET' | 'LENGTH';
+
+export function GetReorderedGoods(
+  goods: string[],
+  sortType: SortType,
+  isReversed: boolean,
+) {
```

Comment:
it`s not ok to return jsx like this in separate function, use this function to reorder items and return reordered list, then map it inside of App component

and why do you need export here?